cmd/api-response.go

```diff
@@ -532,9 +541,12 @@ func generateListObjectsV1Response(bucket, prefix, marker, delimiter, encodingTy
 	data.IsTruncated = resp.IsTruncated
 
 	prefixes := make([]CommonPrefix, 0, len(resp.Prefixes))
-	for _, prefix := range resp.Prefixes {
+	for _, cprefix := range resp.Prefixes {
 		var prefixItem = CommonPrefix{}
-		prefixItem.Prefix = s3EncodeName(prefix, encodingType)
+		prefixItem.Prefix = s3EncodeName(cprefix, encodingType)
+		if cprefix == slashSeparator {
```

Comment:
s/cprefix/prefix ?

Reply:
it is `cprefix` @vadmeste since that is the `for loop`